docs/installation.rst

```diff
@@ -22,9 +30,9 @@ Set Up Multinet
 
    and then launch the Docker container using::
 
-       $ ARANGO_DATA=~/.local/multinet/arango docker-compose up
+       $ ARANGO_DATA=~/.local/multinet/arango docker-compose up -d
 
-3. Inspect the ``.env`` file, which contains a few useful environment variable
+5. Inspect the ``.env`` file, which contains a few useful environment variable
```

Comment:
Since the .env file is more of an important configuration device now, I think this step should be moved above the "run docker-compose" step.

Reply:
Will do